quay-enterprise/aci-signing-keys.md

```diff
@@ -0,0 +1,43 @@
+# Generating signing keys for ACI conversion
+
+## Download the generation script and config
+
+Download the files [aci-signing-key-batch](aci-signing-key-batch) and [generate-signing-keys.sh](generate-signing-keys.sh) next to your configuration directory.
```

Comment:
Download the configuration file [`aci-signing-key-batch`]() and the [`generate-signing-keys.sh`]() script to your configuration directory.